src/analysis.rs

```diff
@@ -96,8 +96,8 @@ impl PerCrateAnalysis {
 }
 
 impl Analysis {
-    pub fn new() -> Analysis {
-        Analysis {
+    pub fn new() -> Self {
+        Self {
```

Comment:
I don't like this change - it adds an annoying level of indirection for the reader